### PR TITLE
fix: bubble up validation errors

### PR DESCRIPTION
**Pull request by refcell**

### CODE REVIEW

**Comment by refcell** [comment]
* **#388** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/kona/388?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @refcell and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by clabby — APPROVED** [review]
LGTM. For some of the logs in the `BatchQueue`, I don't like that we've demoted them to `debug`. Left comments on those.

File: crates/derive/src/stages/attributes_queue.rs [hunk 1]
```diff
@@ -143,7 +143,7 @@ where
         attributes.no_tx_pool = true;
         attributes.transactions.extend(batch.transactions);
 
-        info!(
+        debug!(
```
Comment (clabby) [inline crates/derive/src/stages/attributes_queue.rs:146]: This one too - why `debug` now?

Comment (refcell) [reply crates/derive/src/stages/attributes_queue.rs:146]: It's very verbose when running trusted sync and I don't really consider these logs as info - it's really a trace or a debug. It provides introspection if you want to increase the verbosity level, but it doesn't provide useful information to downstream consumers of the pipeline I think. It just clutters logs.

**Pull request merged**